test/test_get.py

```diff
@@ -0,0 +1,141 @@
+import unittest2
+from test.helpers.httpretty_extension import httpretty
+from test.helpers.merged_datasets_helper import setupDatasetsTest
+import pandas
+import numpy
+from mock import patch, call
+from quandl.model.dataset import Dataset
+from quandl.model.merged_dataset import MergedDataset
+from quandl.get import get
+from quandl.api_config import ApiConfig
+
+
+class GetSingleDatasetTest(unittest2.TestCase):
+    @classmethod
+    def setUp(self):
+        # ensure api key is not set
+        ApiConfig.api_key = None
+        setupDatasetsTest(self, httpretty)
+
+    @classmethod
+    def tearDownClass(cls):
+        httpretty.disable()
+        httpretty.reset()
+
+    def test_returns_pandas_by_default(self):
+        result = get('NSE/OIL')
+        self.assertIsInstance(result, pandas.core.frame.DataFrame)
+
+    def test_returns_pandas_when_requested(self):
+        result = get('NSE/OIL', returns='pandas')
+        self.assertIsInstance(result, pandas.core.frame.DataFrame)
+
+    def test_returns_numpys_when_requested(self):
+        result = get('NSE/OIL', returns='numpy')
+        self.assertIsInstance(result, numpy.core.records.recarray)
+
+    def test_sets_api_key_using_authtoken_arg(self):
+        get('NSE/OIL', authtoken='api_key')
+        self.assertEqual(ApiConfig.api_key, 'api_key')
+
+    def test_sets_api_key_using_api_key_arg(self):
```

Comment:
By default a user should be able to set their api-key in the config and not require it in the kwargs. Maybe we should add a test for that.